SecurityPkg/Include/Library/Tpm2CommandLib.h

```diff
@@ -1241,4 +1241,21 @@ Tpm2PcrReadForActiveBank (
   OUT     TPML_DIGEST  *HashList
   );
 
+/**
+  MU_CHANGE
+  Check if all hash algorithms supported in HashAlgorithmMask are
+  present in the DigestList.
+
+  @param DigestList         Digest list
+  @param HashAlgorithmMask  Bitfield of allowed hash algorithms.
+
+  @retval TRUE  All hash algorithms present.
+  @retval FALSE Some hash algorithms not present.
+**/
+BOOLEAN
+IsDigestListInSyncWithHashAlgorithmMask (
+  IN TPML_DIGEST_VALUES  *DigestList,
+  IN UINT32              HashAlgorithmMask
+  );

```

Comment:
This will line up cleaner in a future diff if "MU_CHANGE" was outside the function comment block.